pages/recognized-delegates/finances/reports/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { CURRENT_ENVIRONMENT } from '@ses/config/endpoints';
+import RecognizedDelegatesContainer from '@ses/containers/recognized-delegates/recognized-delegates';
+import { featureFlags } from 'feature-flags/feature-flags';
+import React from 'react';
+import type { NextPage } from 'next';
+
+const RecognizedDelegates: NextPage = () => {
+  if (!featureFlags[CURRENT_ENVIRONMENT].FEATURE_RECOGNIZED_DELEGATES) {
+    return <div />;
+  }
```

Comment:
This is not required as you can just return a 404 in the SSR method

Reply:
done